docs/src/operate/c8y/custom-fragments.md

```diff
@@ -120,3 +122,4 @@ tedge mqtt pub --retained  te/device/main///twin/c8y_Hardware ''
 
 For information on which fragments Cumulocity supports please see the
 [Cumulocity API docs](https://cumulocity.com/docs/device-integration/fragment-library/).
+
```

Comment:
Would be nice to remove this unnecessary change (I guess it was added by some auto-formatter?)